docs/dev_guide/diagnostic_ref/dpct1056.rst

```diff
@@ -22,7 +22,63 @@ will be removed.
 Suggestions to Fix
 ------------------
 
-If this variable is only used in host code, ignore this warning.
+Command:

```

Comment:
If this sample command is the command use to migrate the example CUDA code to the example migrated SYCL code, it needs introduction/context.  If there is anything of note about the command, it should be explained. For example: 

"For example, this original CUDA code: .."
"is migrated using the following migration command: [example command] "
"which results in the following migrated SYCL code: ..."
"If the variable "const_a" is only used in host code, just ignore the DPCT1056 warning. For example: .."
"If the variable "const_a" is used in both host and device code, rewrite the migrated code. For example: ... "

Reply:
addressed